metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py

```diff
@@ -157,21 +157,19 @@ def get_projects(self) -> List[BigqueryProject]:
                 # Also project.list returns max 50 projects per page.
                 # Assuming list_projects method internally not adding any limit in requests call,
                 # externally we are adding limit in requests call per second.
+                rate_limiter = RateLimiter(max_calls=2, period=1)
```

Comment:
```suggestion
                rate_limiter = RateLimiter(max_calls=2, period=60)
```

Reply:
Quata for projects.list is 2 requests per second not per minute.
I know earlier I added the sleep of 30 sec but it was incorrect.